### DISCUSSION TITLE
Angular Swiper with Thumbs

### DISCUSSION THREAD

**Discussion by sukhbains** [nolimits4web/swiper#4234] [Q&A]
Hey guys,
Love that Swiper JS can be used in Angular!
Im trying to get it to work with thumbnails. On the website there isn't any guide to, so I thought I post a discussion. 

I used an example from the webiste and modifided the code:
https://stackblitz.com/edit/swiper-angular-example-g9nrbi?file=src/app/app.component.ts

Any help would be greatly appreciated.

Thanks!

**Comment by vltansky** [comment]
Hey, right Angular didn't support thumbs & controllers. This was in my TODO list :)
This PR will add support: https://github.com/nolimits4web/swiper/pull/4235

**Reply by sukhbains** [reply]
Legend! Thanks for the reply!
